### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#286] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319548-14bbd207-73ac-41b8-9f9f-fca575c02027.png)

**Comment by LuginiAndrea** [comment]
Inoltre, per ogni 4 numeri in $\mathbf{Z}$, se il loro prodotto è identico, vuol dire che o le coppie sono identiche (al massimo sono "sfasate", ovvero $a = d \wedge b = c$, oppure le coppie sono opposte ($a = -c, b = -d$, in quanto i $-1$ si annullano (anche in questo caso possono essere "sfasate").
Ne consegue che le classi di equivalenza di:
$\left(1,1\right) = \left(1,1\right), \left(-1, -1\right)$
$\left(1,2\right) = \left(1,2\right), \left(2,1\right), \left(-1,-2\right), \left(-2,-1\right)$
Analogamente le altre

**Comment by simyKek** [comment]
Vorrei segnalare un errore nella prima parte dell'esercizio svolta da LuginiAndrea:
Riflessiva: (a,b)R(a,b) => ab = ba che, usando la proprietà commutativa della moltiplicazione, diviene ab = ab.

Simmetria: (a,b)R(c,d) => ad = bc e (c,d)R(a,b) => cb = da che, come sopra, usando la commutatività, diviene bc = ad

Transitività: (a,b)R(c,d) e (c,d)R(w,z) si risolvono come sopra. Per ottenere transitività, però, farei uno step aggiuntivo:
                  Con prima relazione, si ha che ad = bc; isolando la c, si ha che c = ad/b.
                  Se sostituiamo c nella seconda relazione, otteniamo che: cz = dw => adz/b = dw; le due d si semplificano e, portando a destra b tramite moltiplicazione da ambe le parti, viene az = bw che implica (a,b)R(w,z)

(Chiederei a qualcuno più capace di me di controllare correttezza e se ho eventualmente fatto errori. Inoltre, mi scuso se sto pubblicando queste cose solo un anno dopo, ma sto dando esame ora :D )
